source/leuven/slurm_specifics.rst

```diff
@@ -0,0 +1,172 @@
+.. _leuven_slurm_specifics:
+
+Site-specific Slurm info
+========================
+
+While the :ref:`Running jobs in Slurm <running jobs>` pages provide generic
+information regarding Slurm, there are additional points to consider when
+when using Slurm on Tier-2 clusters hosted at KU Leuven.
+
+
+Compute credits
+---------------
+When submitting a job, you need to provide a valid Slurm credit account holding
+enough compute credits for the job using the ``-A/--account`` option.
+For more information, please consult the following pages:
+
+.. toctree::
+   :maxdepth: 1
+
+   ./credits
+   ./slurm_accounting
+
+
+.. _leuven_job_shell:
+
+Job shell
+---------
+For batch jobs we strongly recommend to use ``#!/bin/bash -l`` as the shebang
```

Comment:
*Optional*: mention that this is a lowercase L, not an uppercase i. I have seen confusion about this.

Reply:
Indeed best to make this clearer (-> 37ea3f76d2685628d14c84b5018376cf796bdc79)